src/money-formatter.js

```diff
@@ -2,11 +2,18 @@
 import currencyFormatData from 'currency-format';
 import { formatDigits, isNumeric } from './utils';
 
-const getCurrencyData = (code) =>
+const getCurrencyData = code =>
   currencyFormatData[code.toUpperCase()];
 
-export const formatSimple = (currencyName, amount, fractionSize) =>
-  `${formatDigits(amount, fractionSize)} ${currencyName}`;
+export const formatSimple = (currencyName, amount, fractionSize) => {
+  const result = [];
+  result.push(formatDigits(amount, fractionSize));
+  if (currencyName && currencyName !== '') {
```

Comment:
Checking for empty string is redundant, since empty string is already a falsy value.

Reply:
:shakes fist at the existence of `==`:

![image](https://user-images.githubusercontent.com/871756/31650990-641a630c-b2df-11e7-96b7-febf13f3f2fc.png)